src/network/networkd-address.c

```diff
@@ -723,10 +732,10 @@ int config_parse_address(const char *unit,
         if (streq(section, "Network")) {
                 /* we are not in an Address section, so treat
                  * this as the special '0' section */
-                section_line = 0;
-        }
+                r = address_new_static(network, "Network", 0, &n);
```

Comment:
Is this distinction still necessary? Would it be possible to use the filename and section_line of the "Network" section as arguments to `address_new_static` and get rid of this `if`?